service/collector/helper.go

```diff
@@ -0,0 +1,18 @@
+package collector
+
+import (
+	"time"
+
+	"github.com/giantswarm/microerror"
+)
+
+func convertToTime(datetime string) (time.Time, error) {
```

Comment:
Let's put that in the file where it is used. It is not getting better with this `helper.go` file. 😄